package.json

```diff
@@ -10,7 +10,7 @@
     "start": "next start",
     "lint": "next lint --fix",
     "validate-md": "node scripts/validateMarkdown.js",
-    "test": "percy exec -- cypress run",
+    "test": "cypress run --spec 'cypress/e2e/**/*.cy.ts,!cypress/e2e/weekly_schedule/**/*.cy.ts'",
```

Comment:
Was the decision to remove`percy` intentional?

Reply:
Not a blocker > Have you considered splitting the test scripts into multiple commands for different contexts (e.g., `test:all`, `test:critical`, `test:weekly_schedule`). This can help in running specific sets of tests based on the situation.

```
"scripts": {
  "test": "cypress run",
  "test:all": "cypress run --spec 'cypress/e2e/**/*.cy.ts'",
  "test:critical": "cypress run --spec 'cypress/e2e/critical/**/*.cy.ts'",
  "test:weekly_schedule": "cypress run --spec 'cypress/e2e/weekly_schedule/**/*.cy.ts'"
}
```

Reply:
Another thing we could do is consider using environmental variables.

```
"scripts": {
  "test": "cypress run --spec 'cypress/e2e/**/*.cy.ts${TEST_SPEC_EXCLUDE}'"
}
```

```
const testSpecExclude = process.env.TEST_SPEC_EXCLUDE;
```

Reply:
Nice! I like that